Formula/git-sh.rb

```diff
@@ -17,7 +17,7 @@ class GitSh < Formula
     sha256 "1b9aa141f32145516db62304dda799611e1fc35ec57275ee75bf566325a6bfa5" => :mavericks
   end
 
-  disable! date: "2020-12-08", because: :unmaintained
+  disable! date: "2017-11-08", because: :unmaintained
```

Comment:
We should probably leave this one unchanged, despite it being unmaintained for much longer. With the new date, @BrewTestBot will try to delete this formula the day after this PR is merged. This defeated the purpose of the disable period.